meshroom/nodes/aliceVision/SketchfabUpload.py

```diff
@@ -0,0 +1,282 @@
+        self._cb_kwargs.update({
+            'size'    : self._len,
+            'progress': self._progress
+        })
+        if self._callback:
+            try:
+                self._callback(*self._cb_args, **self._cb_kwargs)
+            except Exception as e: # catches exception from the callback
+                self._cb_kwargs['logManager'].logger.warning('Error at callback: {}'.format(e))
+
+        if self._stopped():
+            raise RuntimeError('Node stopped by user')
+        return chunk
+
+def progressUpdate(size=None, progress=None, logManager=None):
+    if not logManager.progressBar:
+        logManager.makeProgressBar(size, 'Upload progress:')
+
+    logManager.updateProgressBar(progress)
+
+class SketchfabUpload(desc.Node):
+    size = desc.DynamicNodeSize('inputFiles')
+    inputs = [
+        desc.ListAttribute(
+            elementDesc=desc.File(
+                name="input",
+                label="Input",
+                description="",
+                value="",
+                uid=[0],
+            ),
+            name="inputFiles",
+            label="Input Files",
+            description="Input Files to export.",
+            group="",
+        ),
+        desc.StringParam(
+            name='apiToken',
+            label='API Token',
+            description='Get your token from https://sketchfab.com/settings/password',
+            value='',
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='title',
+            label='Title',
+            description='Title cannot be longer than 48 characters.',
+            value='',
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='description',
+            label='Description',
+            description='Description cannot be longer than 1024 characters.',
+            value='',
+            uid=[0],
+        ),
+        desc.ChoiceParam(
+            name='license',
+            label='License',
+            description='License label.',
+            value='CC Attribution',
+            values=['CC Attribution',
+                    'CC Attribution-ShareAlike',
+                    'CC Attribution-NoDerivs',
+                    'CC Attribution-NonCommercial',
+                    'CC Attribution-NonCommercial-ShareAlike',
+                    'CC Attribution-NonCommercial-NoDerivs'],
+            exclusive=True,
+            uid=[0],
+        ),
+        desc.ListAttribute(
+            elementDesc=desc.StringParam(
+                name='tag',
+                label='Tag',
+                description='Tag cannot be longer than 48 characters.',
+                value='',
+                uid=[0],
+            ),
+            name="tags",
+            label="Tags",
+            description="Maximum of 42 separate tags.",
+            group="",
+        ),
+        desc.ChoiceParam(
+            name='category',
+            label='Category',
+            description='Maximum of 2 categories.',
```

Comment:
Exclusive is set to True, so in all case, we can choose only one category.
So the description or the "exclusive" mode should be adjusted.